docs/using-radicle/viewing-and-merging-patches.md

```diff
@@ -0,0 +1,59 @@
+---
+id: viewing-and-merging-patches
+title: Viewing and merging patches
+---
+
+To collaborate on a Radicle project, you need to know how to view and merge
+patches from other people. To view and merge patches from a contributor, you
+first need to add them as a remote to your project. If you haven't done so,
+follow the instructions in [Tracking & viewing contributions][tv] section.
+
+If you can see a peer's patches after adding them as a remote, they have been
+successfully replicated to your machine. If you have added a peer as a remote
+and can't see their changes, head over to our [FAQ][fq] section.
+
+## Viewing patches
+
+Now that you're tracking the contributor, you can navigate to the `Patches` tab
+on your project and see any patches that contributor has created.
+
+![Patches][pa]
+
+You can then click on the patch that you're interested in and get a detailed
+view. You'll see the title, description, commits, and the branches it's
+comparing. If you're the maintainer, you'll also see two buttons: `Checkout` and
+`Merge`.
+
+
+![PatchDetail][pd]
+
+## Checking out & merging a patch
+
+ If you're the maintainer, you can easily checkout a patch by clicking
+ `Checkout` and copying the command:
+
+`git fetch rad remotes/hynadcs...s3gxjw/tags/radicle-patch/change-styles:tags/radicle-patch/coolprogrammer9000/change-styles`
```

Comment:
This is still missing the actual checkout command
```
git checkout tags/radicle-patch/coolprogrammer9000/change-styles
```
I’ll also update the patch instructions.

Reply:
I added this in https://github.com/radicle-dev/radicle-upstream/pull/1682/commits/ac47a5eaa96d4dc61df57d2e98e9ba595b844181